src/api/server/ServerApi.ts

```diff
@@ -481,8 +481,7 @@ export default class ServerApi {
     removeSync(recipeTempDirectory);
     removeSync(join(recipesDirectory, recipeId, 'recipe.tar.gz'));
 
-    // TODO: This is a temporary fix to remove svg icons from the user AppData. This should be removed after some versions.
-    debug('Removing default icon', defaultIcon);
+    // TODO: This is a temporary fix to remove svg icons from the user AppData. This should be removed after versions of all recipes have been bumped up
```

Comment:
@vraravam in reality this code should **not** be removed to ensure that user local recipes (custom recipes made by users) don't lose functionality. If we keep the next line we ensure that everything works properly 😁

I propose we delete this TODO comment as we have nothing to do 🙏

Reply:
yes, i didn't think of custom recipes. We can remove it in the next PR whenever that happens